code-env/python/spec/requirements.txt

```diff
@@ -8,4 +8,5 @@ opencv-python==4.7.0.72; python_version >= '3.10'
 deskew==0.10.33
 torch==1.11.0; python_version >= '3.10'
 torch==1.9.1; python_version <= '3.9'
-easyocr==1.7.0
\ No newline at end of file
+easyocr==1.7.0
+packaging==21.3
```

Comment:
Because they forgot.
`packaging` is a dependency of `scikit-image` which now has `packaging` as dependency but the last working version of `scikit-image` for python 3.6 doesn't have `packaging` as dependency.
No need to have a filter, it won't change much (the real problem is easyocr that doesn't have any fixed version requirements).